.github/ISSUE_TEMPLATE/funding_application.yml

```diff
@@ -0,0 +1,235 @@
+          - If no, recommend to route through a Supplemental Funding Initiative?
+  - type: markdown
+    attributes:
+      value: |
+        **Communication of Final Decision:**
+        - Complete like a CFP. Notify in writing if it was accepted or not accepted. Put out a list of accepted proposals (like the events schedule).
+  - type: markdown
+    attributes:
+      value: "Problem statement - This section aims to set the stage and boundaries
+        for the problem you are trying to solve. This is not a place to propose
+        solutions, but to define the problem and its impact."
+  - type: input
+    id: Problem
+    attributes:
+      label: Problem Statement
+      description: What is the problem?
+      placeholder: null
+    validations:
+      required: true
+  - type: input
+    id: impact
+    attributes:
+      label: Who does this affect?
+      description: Who does this problem impact, and for whom is this solution?
+      placeholder: null
+    validations:
+      required: true
+  - type: input
+    id: prior
+    attributes:
+      label: Have there been previous attempts to resolve the problem?
+      description: Why is this approach different, or why didn't the other approaches work?
+      placeholder: null
+    validations:
+      required: true
+  - type: input
+    id: why
+    attributes:
+      label: Why should it be tackled now and by this TI?
+      description: Is this the right place and time for this funding initiative?
+      placeholder: null
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: Proposed work and other work considered - The goal here is to assess the
+        feasibility of the proposed work and to understand the alternatives that
+        were considered. Such as can be done now and is in scope with the
+        current funds.
+  - type: textarea
+    id: inputs
+    attributes:
+      label: Give an idea of what is required to make the funding initiative happen
+      placeholder: null
+    validations:
+      required: true
+  - type: textarea
+    id: deliver
+    attributes:
+      label: What is going to be needed to deliver this funding initiative?
+      placeholder: null
+    validations:
+      required: true
+  - type: textarea
+    id: tools
+    attributes:
+      label: Are there tools or tech that still need to be produced to facilitate the
+        funding initiative?
+      placeholder: null
+    validations:
+      required: false
+  - type: textarea
+    id: requirements
+    attributes:
+      label: Give a summary of the requirements that contextualize the costs of the
+        funding initiative
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: Covering the planning phase, this next section should provide a a
+        relatively detailed plan of how work will start on the project. This
```

Comment:
Since "Project" has a particular meaning pertaining to TIs, I'm a little concerned the usage of the term here may cause some confusion. Above sections use "funding initiative", which I think might be a better fit here as well.